registries/docker-daemon/cleanup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# nothing to do
```

Comment:
This should minimally stop/delete the registry process running in the daemon. Ideally, it would also remove the k8s service and endpoint.